### DISCUSSION TITLE
Upgrading to quasar 2- global property undefined

### DISCUSSION THREAD

**Discussion by arunima-nair** [quasarframework/quasar#13569] [Quasar App Extensions] [answered]
Teams,
I am facing issue with global property undefined after upgrading to quasar 2.
below shows declaration 
![image](https://user-images.githubusercontent.com/37262910/170965498-b64df71e-8fdb-42dd-8295-c9cf95b58def.png)
When i call  another component it showing undefined error as below:
![image](https://user-images.githubusercontent.com/37262910/170965935-e00483ed-d01b-4eff-ab8b-b7b7a5fae7ab.png)
![image](https://user-images.githubusercontent.com/37262910/170966228-eabf8a5c-0f74-4fba-8dd0-93a13f35ccc3.png)
![image](https://user-images.githubusercontent.com/37262910/170966587-9ce1d455-eada-47d1-9a9e-addb697220e5.png)

**Comment by yusufkandemir** [comment, marked as answer]
You shared the "declaration" of something, but we don't know what it is. If it's a boot function, you should use the `app` from the callback's parameter. `export default ({ app }) => {`. Please share the full code.

**Reply by arunima-nair** [reply]
![image](https://user-images.githubusercontent.com/37262910/171587510-ef9c10c1-b3c0-4980-bcc2-419dac2e94c3.png)
By changing code like this my error resolved.
